packages/app/src/sandbox/eval/transpilers/vue/v2/component-normalizer.js

```diff
@@ -13,19 +13,22 @@ module.exports = function normalizeComponent(
   scopeId,
   moduleIdentifier /* server only */
 ) {
-  var esModule;
-  var scriptExports = (rawScriptExports = rawScriptExports || {});
+  let scriptExports = (rawScriptExports = rawScriptExports || {});
 
   // ES6 modules interop
-  var type = typeof rawScriptExports.default;
-  if (type === 'object' || type === 'function') {
-    esModule = rawScriptExports;
-    scriptExports = rawScriptExports.default;
-  }
+  //   let type = typeof rawScriptExports.default;
```

Comment:
Is this even necessary? Doesn't babel handle this for imports anyway?

Reply:
In case it is, I changed the logic in the comment to reassign other exports to the object or function it returns but it's not super reliable